test/orchard/inspect_test.clj

```diff
@@ -597,16 +597,29 @@
                  (inspect/next-sibling)
                  (inspect/next-sibling)
                  :value)))
-    (is (= 41 (-> long-vector
+    (is (= 45 (-> long-vector
                   inspect
-                  (inspect/set-page-size 6)
+                  (inspect/set-page-size 3)
                   (inspect/down 40)
                   (inspect/next-sibling)
                   (inspect/next-sibling)
                   (inspect/next-sibling)
                   (inspect/next-sibling)
                   (inspect/next-sibling)
                   (inspect/next-sibling)
+                  :value))))
+  (testing "next-sibling doesn't fall beyond the last element."
```

Comment:
Please also exercise `next-sibling doesn't fall beyond the last element` for:

* small vector
  *  for a self-evident test
* a range

Reply:
Made both the changes.